### DISCUSSION TITLE
Add object count in map selector

### DISCUSSION THREAD

**Discussion by uwuclxdy** [ppy/osu#26462] [Ideas]
Basically the title; the object count should be displayed next to slider count, circle count, etc..
![image](https://github.com/ppy/osu/assets/37777261/79e715ca-72c5-43d4-a0ac-2465975d6f07)

Also, since the new update, next to that there should also be a max combo counter, so it's not needed to go check it in a recent score.

**Comment by Gabixel** [comment]
The entire carousel UI will be redesigned, so we have to wait for that first I think.

I agree with the max combo being visible (there was also an attempt to [show it in each leaderboard score](https://github.com/ppy/osu/pull/26227)).

I think having the split objects count is the best, I don't really see the strategy/point of having the sum visible, it doesn't give you that much information about the beatmap (i.e. what you'll have to deal with). Maybe showing it as a tooltip message is enough if you want it.

**Comment by peppy** [comment]
To confirm, all this would be doing is saving you the mental math of adding three numbers together? I'm not sure it deserves the space if so. Maybe we need a better display to show the object counts with some kind of inline total.
